python/torch_mlir/dialects/torch/importer/jit_ir/build_tools/abstract_interp_lib_gen.py

```diff
@@ -1026,19 +1026,107 @@ def _get_invocations_for_op_with_tensor_arg_followed_by(*args):
     dtype function instead of using this helper function.
     """
     return [
-    Invocation(NonZeroDTensorWithDtype(torch.float32), *args),
-    Invocation(NonZeroDTensorWithDtype(torch.float64), *args),
-    Invocation(NonZeroDTensorWithDtype(torch.bfloat16), *args),
-    Invocation(NonZeroDTensorWithDtype(torch.int64), *args),
-    Invocation(NonZeroDTensorWithDtype(torch.int32), *args),
-    Invocation(NonZeroDTensorWithDtype(torch.bool), *args),
-    Invocation(ZeroDTensorWithDtype(torch.float32), *args),
-    Invocation(ZeroDTensorWithDtype(torch.float64), *args),
-    Invocation(ZeroDTensorWithDtype(torch.bfloat16), *args),
-    Invocation(ZeroDTensorWithDtype(torch.int64), *args),
-    Invocation(ZeroDTensorWithDtype(torch.int32), *args),
-    Invocation(ZeroDTensorWithDtype(torch.bool), *args),
-]
+        Invocation(NonZeroDTensorWithDtype(torch.float32), *args),
+        Invocation(NonZeroDTensorWithDtype(torch.float64), *args),
+        Invocation(NonZeroDTensorWithDtype(torch.bfloat16), *args),
+        Invocation(NonZeroDTensorWithDtype(torch.int64), *args),
+        Invocation(NonZeroDTensorWithDtype(torch.int32), *args),
+        Invocation(NonZeroDTensorWithDtype(torch.bool), *args),
+        Invocation(ZeroDTensorWithDtype(torch.float32), *args),
+        Invocation(ZeroDTensorWithDtype(torch.float64), *args),
+        Invocation(ZeroDTensorWithDtype(torch.bfloat16), *args),
+        Invocation(ZeroDTensorWithDtype(torch.int64), *args),
+        Invocation(ZeroDTensorWithDtype(torch.int32), *args),
+        Invocation(ZeroDTensorWithDtype(torch.bool), *args),
+    ]
+
+def _get_invocations_for_fp_only_op_with_tensor_arg_followed_by(*args):
```

Comment:
I will be making a PR soon that improves the testing helper functions quite a bit, so if you're working on another set of ops, I would wait until that lands

Reply:
Awesome! I haven't started working on the next task. Will wait for your PR before further development.